### PR TITLE
Update ratingscalc.py

### PR DESCRIPTION
**Pull request by SwatiSuri**

### CODE REVIEW

**Comment by ischo0812** [comment]
Testing Not Working

File: app/src/ratingscalc.py [hunk 1]
```diff
@@ -6,3 +6,21 @@
 #
 # If the list of reviews is empty, return an empty dictionary
 # Make sure that you add type hints to the function paramter and return value
+from Review import review
+from utils import  cal_avg_rating
+
+def cal_avg_rating(reviews: list[review]) -> dict[int, float]:
+    restaurant_rating ={}
+    total_ratings = {}      
+    if reviews == none:
```
Comment (mab105120) [inline app/src/ratingscalc.py:15]: Please be careful: none is not the same as None (with capital N). Also to check whether a variable is None use the built-in isinstance function: isinstance(reviews, None)

File: app/src/ratingscalc.py [hunk 2]
```diff
@@ -6,3 +6,21 @@
...
+       return {}
+    for review in reviews:
+        if review.getrestaurant() in restaurant_rating.keys():
+           restaurant_rating[review.getrestaurant()] += review.getrating()
```
Comment (mab105120) [inline app/src/ratingscalc.py:19]: this should create a dictionary from a restaurant name to a list of the reviews for that resturant. To add to a list use the append function not +=

File: app/src/ratingscalc.py [hunk 3]
```diff
@@ -6,3 +6,21 @@
...
+           total_ratings[review.getrestaurant()] += 0 
+        else:
+            restaurant_rating[review.getrestaurant()] = review.getrating()
+            total_ratings[review.getrestaurant()] = 0
+    for restaurant in restaurant_rating.keys():
+        restaurant_rating[restaurant] = round((restaurant_rating[restaurant])/total_ratings[restaurant],2)
```
Comment (mab105120) [inline app/src/ratingscalc.py:25]: You didn't need to calculate the average explicitly here again. You already created a function in the utils module to compute averages, you should reuse that code here as well. Reuse code as much as possible to reduce redundancy in your code.

**Pull request closed without merging**